src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
+    const filter = Iterators.filter
+end
+
+if VERSION < v"0.5"
+    min(x) = x
+    max(x) = x
```

Comment:
These are just not defined on 0.4. Do you mean that I should change this to `VERSION <= v"0.4"`?